['text':' Owner(s): ["module: dynamo"]','line_number':1,'multiline':False]['text':' reason="XXX: np.dtype() objects not supported"),','line_number':33,'multiline':False]['text':' numpy namespaced dtypes not supported','line_number':43,'multiline':False]